x/concentrated-liquidity/incentives_test.go

```diff
@@ -3535,6 +3528,69 @@ func (s *KeeperTestSuite) TestQueryAndClaimAllIncentives() {
 	}
 }
 
+func (s *KeeperTestSuite) TestClaimAllIncentivesForPosition() {
```

Comment:
reason why this is not tdd is because we aren't testing for the whole function but just one specific function call of the function `updatePoolUptimeAccumulatorsToNow` so this approach was a lot cleaner and simpler